cpp/src/arrow/compute/exec/aggregate_node.cc

```diff
@@ -697,13 +909,18 @@ class GroupByNode : public ExecNode, public TracedNode {
   }
 
   int output_task_group_id_;
+  std::unique_ptr<GroupingSegmenter> segmenter_;
```

Comment:
Please add documentation for the new members

Reply:
Done.

Reply:
Can you point me to where the documentation is?

Reply:
[Here](https://github.com/apache/arrow/pull/34311/files#diff-f53040e5fc284581263737f96238c8a76ef54d13f01e34305c1d1d9de5e6e34bR492-R509) and [here](https://github.com/apache/arrow/pull/34311/files#diff-f53040e5fc284581263737f96238c8a76ef54d13f01e34305c1d1d9de5e6e34bR925-R941).